site/contributing-docs.md

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
This file should be located in https://github.com/projectcontour/contour/tree/main/site/content/resources

Reply:
Add the following front matter at the top of of this document:

```
---
title: Contributing Technical Documentation
layout: page
---
```